### PR TITLE
Release 2.0

### PR DESCRIPTION
**Pull request by btseytlin**

### CODE REVIEW

**Review by btseytlin — COMMENTED** [review]
Reviewed, looks good

File: mindsdb_native/libs/data_sources/mongodb_ds.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+from pymongo import MongoClient
+
+from mindsdb_native.libs.data_types.data_source import DataSource
+
+
+class MongoDS(DataSource):
+    def _setup(self, collection, query=None, host='localhost', port=27017, user='admin',
+               password='123', database='database'):
```
Comment (StpMax) [inline mindsdb_native/libs/data_sources/mongodb_ds.py:9]: i think we should set default password and database to empty string

File: mindsdb_native/libs/phases/type_deductor/type_deductor.py [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+    cast_string_to_python_type,
+    is_foreign_key
+)
 from mindsdb_native.libs.phases.base_module import BaseModule
-from mindsdb_native.libs.helpers.text_helpers import (word_tokenize,
-                                               cast_string_to_python_type,
-                                               is_foreign_key)
 from mindsdb_native.libs.helpers.stats_helpers import sample_data
 
+import flair
```
Comment (StpMax) [inline mindsdb_native/libs/phases/type_deductor/type_deductor.py:25]: unused dependency

File: optional_requirements_extra_data_sources.txt [hunk 3]
```diff
@@ -1,2 +1,4 @@
 boto3 >= 1.9.0
 pg8000 >= 1.15.3
+python-tds >= 1.10.0
```
Comment (StpMax) [inline optional_requirements_extra_data_sources.txt:3]: where it used?

**Review by StpMax — APPROVED** [review]

File: optional_requirements_extra_data_sources.txt (hunk 3 above)
Comment (maximlopin) [reply optional_requirements_extra_data_sources.txt:3]: microsoft-sql datasource

**Review by George3d6 — CHANGES_REQUESTED** [review]
Currently the outlier detection is broken since PR #26 

A fix for this was included *but* it isn't merged yet, we should postpone the release until the tests pass, the PR is reviewed and then merged.

**Comment by George3d6** [comment]
Alright, my complaint I've addressed, all should be fine, merging now.

**Pull request merged**